src/main/java/gov/usgs/ngwmn/model/PCode.java

```diff
@@ -31,9 +31,9 @@
 	private final DepthDatum datum;
 	private final String description;
 	private final Unit unit;
-	private final boolean up;	//true if positive values indicate the direction upwards from the datum.
-	private final boolean unrecognized;	//If we do not recognize the name.
-	private final boolean unspecified; //If the pcode is empty or null.
+	private final boolean up;	        // True if positive values indicate the direction upwards from the datum.
+	private final boolean unrecognized;	// If we do not recognize the name.
+	private final boolean unspecified;  // If the pcode is empty or null.
```

Comment:
I appreciate the differentiation between these boolean properties.  With stats/math calculations, these names help me understand why a condition may be true or false.

Reply:
I suppose it could be an enum. It kinda grew organically as needed. I did not create the 'up' parameter. It should be MediationType really. I pulled that code from the cache project directly.

Reply:
Your suggested change to the `up` param sounds good to me.  Could also make these params part of an enum, I leave it to you to decide.